css/style.css

```diff
@@ -72,4 +72,100 @@ li{
     -ms-border-radius: 50%;
     -o-border-radius: 50%;
 }
-/***** End general *****/
\ No newline at end of file
+/***** End general *****/
+/***** hero *****/
+.hero{
```

Comment:
the whole section content should be aligned to the bottom 
animation is almost touching the end of the screen, and everything else is kinda centred 
![image](https://user-images.githubusercontent.com/44810632/155527624-482ff230-cc2a-4357-884f-0d0549a187bc.png)


here it looks stuck to the top

![image](https://user-images.githubusercontent.com/44810632/155527671-2741e282-198e-4eb4-9acb-2d8e54289418.png)

this has nothing to do with merge or other branches since the nav is already positioned fixed